code-challenges.js

```diff
@@ -6,63 +6,133 @@
-const fruit3 = "cherry"
-const fruit4 = "kiwi"
 // Expected outcome: "cherry"
 
 // --------------------1) Create a function that takes in a number and determines if the given number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below.
 
 // Pseudo code:
-
-const temp1 = 42
+// Create declarations and parameters, where temp is the number as it relates to 212 degrees F
+// Create if, else if statements along with greater/lesser/strictly equal symbols to determine below/at/boiling temps based on variables given
+// Run the function using console.log to verify accuracy of function
+
+
+// const temp1 = 42
+// const temp2 = 350
+// const temp3 = 212
+
+// const boilingPoint = (temp) => {
+//     if(temp < 212)
+//         return `${temp} is below boiling point.`
+//     else if(temp > 212)
+//         return`${temp} is above boiling point.`
+//     else if(temp === 212)
+//         return `${temp} is at boiling point.`
+// }
+// console.log(boilingPoint(temp1))
+// console.log(boilingPoint(temp2))
+// console.log(boilingPoint(temp3))
+//     42 is below boiling point.
+//     350 is above boiling point.
+//     212 is at boiling point.
 // Expected output: "42 is below boiling point"
-
-const temp2 = 350
 // Expected output: "350 is above boiling point"
-
-const temp3 = 212
 // Expected output: "212 is at boiling point"
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 // Pseudo code:
+// Create declarations of Padres World Series runs as arrays
+// Create concatentation of array variables
+// Run using console.log with .length to determine length of combined arrays
 
-const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
-const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+// const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
+// const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+// const padres1984And1998WSRuns = padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns)
+
+// console.log(padres1984And1998WSRuns.length)
+// 9
 // Expected output: 9
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
 // Pseudo code:
+// In console.log, use use Mutators to modify output
+// Use .split to turn string into array
+// Use .reverse to reverse array
+// Use .join to combine elements in array
+// Run console.log with .split("").reverse("").join("")
+
+// const currentCohort = "Alpha 2023"
 
-const currentCohort = "Alpha 2023"
+// console.log(currentCohort.split("").reverse("").join(""))
+// 3202 ahplA
 // Expected output: "3202 ahplA"
+// *Feedback request* "I know that I need to restructure the data type in the declared variable with .split.  I need a little more clarification on the 'why'.  Is it because I need to change the string into an array first in order for .reverse to be considered a function?"
```

Comment:
Great question! .reverse is a method that doesn't work on strings in JavaScript. You are right in that you have to change it into an array using .split. Good job!